### ISSUE TITLE
Notifications happen quite often after restart

### ISSUE THREAD

**Issue by roymckenzie** [elfenware/badger#4]
I installed this last night. I love the concept. I really do need to pay attention to the rest of my body while sitting in front of my machine for several hours each day.

I forgot about the app because I never received any notifications until I restarted my machine about an hour ago today.

Once I restarted, I began receiving notifications and quite frequently. Sometimes the same notification a couple of times in a row.

One solution I was thinking of was to reset all notifications after any period of inactivity greater than 20 minutes. Once a user begins interacting with the computer again, recreate the notifications. 

Thanks again for a good app with a laudable goal.

**Comment by dar5hak** [comment]
@roymckenzie glad you liked the idea.

I'm aware of the repeat notifications issue, and have fixed it in v1.0.3, which is being curated at the time of writing, so it won't be long before you receive it! 😄

As for the frequency of notifications, I have set hard intervals for each notification type. For example, the reminder to blink eyes is sent every 20 minutes. You can check `src/Application.vala`and scroll down to the bottom to see the durations.

If you think some of those are too frequent, I'd love your feedback.

As for stopping for user inactivity, I can't think of a way to detect that (still very new to GTK+ development), but will research and see if it's possible to implement.

Thank you again for your inputs. 😊

**Comment by Hedronmx** [comment]
Maybe even add a custom timer

**Comment by roymckenzie** [comment]
@dar5hak I look forward to the new release! I did some research on detecting system idle-ness. It is a difficult problem. The folks over at SETI had a problem with it, but I am looking around to help find a solution. I found [this piece of code](https://github.com/sudharsh/openmoko-gsoc2008/blob/a14425e0f282ec798b25388d68d6f54c908b2326/fsod/src/subsystems/Device/idlenotifier/idlenotifier.vala#L69) that uses `ODeviced` to listen to device statuses. Not sure if that could be used here. I will look into it more.
